src/test/data/DemoData.js

```diff
@@ -2940,7 +2940,7 @@ export const trendList = [
 
 export const infoObject = {
   composite: {
-    title: 'Composite',
+    title: 'Composite Score',
```

Comment:
That's not going to reflect in the real data. It'll just read `Composite - Composite`. @anthem123 would have to change what is being sent.

Reply:
SO should i get @anthem123 to show me?